src/main/java/com/alipay/oceanbase/hbase/util/OHConnectionConfiguration.java

```diff
@@ -0,0 +1,80 @@
+package com.alipay.oceanbase.hbase.util;
+
+import org.apache.hadoop.classification.InterfaceAudience;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.hbase.HConstants;
+
+@InterfaceAudience.Private
+public class OHConnectionConfiguration {
+    public static final String WRITE_BUFFER_SIZE_KEY     = "hbase.client.write.buffer";
```

Comment:
why implement OHConnectionConfiguration instead of use hbase's configuration directly?

Reply:
The class aims to improve performance by caching connection-related configuration values. It avoids frequent calls to `conf.getXXX()` methods every time classes like `HTable` are instantiated, thus reducing CPU usage and enhancing performance.